File: clients/csharp/src/Swagger/Client/Model/trade.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Swagger.Client.Model {
  public class Trade {
    public DateTime? timestamp { get; set; }

    public string symbol { get; set; }

    public string side { get; set; }

    public double? size { get; set; }

    public double? price { get; set; }

    public string tickDirection { get; set; }

    public string trdMatchID { get; set; }

    public double? grossValue { get; set; }

    public double? homeNotional { get; set; }

    public double? foreignNotional { get; set; }

    public double? id { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Trade {\n");
      sb.Append("  timestamp: ").Append(timestamp).Append("\n");
      sb.Append("  symbol: ").Append(symbol).Append("\n");
      sb.Append("  side: ").Append(side).Append("\n");
      sb.Append("  size: ").Append(size).Append("\n");
      sb.Append("  price: ").Append(price).Append("\n");
      sb.Append("  tickDirection: ").Append(tickDirection).Append("\n");
      sb.Append("  trdMatchID: ").Append(trdMatchID).Append("\n");
      sb.Append("  grossValue: ").Append(grossValue).Append("\n");
      sb.Append("  homeNotional: ").Append(homeNotional).Append("\n");
      sb.Append("  foreignNotional: ").Append(foreignNotional).Append("\n");
      sb.Append("  id: ").Append(id).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: clients/java/jersey/src/main/java/com/wordnik/client/model/trade.java
package com.wordnik.client.model;

import java.util.Date;
public class Trade {
  private Date timestamp = null;
  private String symbol = null;
  private String side = null;
  private Double size = null;
  private Double price = null;
  private String tickDirection = null;
  private String trdMatchID = null;
  private Double grossValue = null;
  private Double homeNotional = null;
  private Double foreignNotional = null;
  private Double id = null;
  public Date getTimestamp() {
    return timestamp;
  }
  public void setTimestamp(Date timestamp) {
    this.timestamp = timestamp;
  }

  public String getSymbol() {
    return symbol;
  }
  public void setSymbol(String symbol) {
    this.symbol = symbol;
  }

  public String getSide() {
    return side;
  }
  public void setSide(String side) {
    this.side = side;
  }

  public Double getSize() {
    return size;
  }
  public void setSize(Double size) {
    this.size = size;
  }

  public Double getPrice() {
    return price;
  }
  public void setPrice(Double price) {
    this.price = price;
  }

  public String getTickDirection() {
    return tickDirection;
  }
  public void setTickDirection(String tickDirection) {
    this.tickDirection = tickDirection;
  }

  public String getTrdMatchID() {
    return trdMatchID;
  }
  public void setTrdMatchID(String trdMatchID) {
    this.trdMatchID = trdMatchID;
  }

  public Double getGrossValue() {
    return grossValue;
  }
  public void setGrossValue(Double grossValue) {
    this.grossValue = grossValue;
  }

  public Double getHomeNotional() {
    return homeNotional;
  }
  public void setHomeNotional(Double homeNotional) {
    this.homeNotional = homeNotional;
  }

  public Double getForeignNotional() {
    return foreignNotional;
  }
  public void setForeignNotional(Double foreignNotional) {
    this.foreignNotional = foreignNotional;
  }

  public Double getId() {
    return id;
  }
  public void setId(Double id) {
    this.id = id;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Trade {\n");
    sb.append("  timestamp: ").append(timestamp).append("\n");
    sb.append("  symbol: ").append(symbol).append("\n");
    sb.append("  side: ").append(side).append("\n");
    sb.append("  size: ").append(size).append("\n");
    sb.append("  price: ").append(price).append("\n");
    sb.append("  tickDirection: ").append(tickDirection).append("\n");
    sb.append("  trdMatchID: ").append(trdMatchID).append("\n");
    sb.append("  grossValue: ").append(grossValue).append("\n");
    sb.append("  homeNotional: ").append(homeNotional).append("\n");
    sb.append("  foreignNotional: ").append(foreignNotional).append("\n");
    sb.append("  id: ").append(id).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

